crypto/src/key/key_holder.rs

```diff
@@ -13,16 +13,23 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+use crate::key::rschnorr::{MLPrivateKey, MLPublicKey};
 use serialization::{Decode, Encode};
 
 use super::rschnorr::{MLRistrettoPrivateKey, MLRistrettoPublicKey};
 
 #[derive(Debug, PartialEq, Eq, Clone, Decode, Encode)]
 pub enum PrivateKeyHolder {
+    #[codec(index = 0)]
```

Comment:
Thank you for adding indexes! Please add it to any enum that needs to be encoded if we forget.